src/roiextractors/extractors/numpyextractors/numpyextractors.py

```diff
@@ -50,10 +50,10 @@ def __init__(
         self._channel_names = channel_names
 
         (
-            self._num_channels,
             self._num_frames,
             self._size_x,
             self._size_y,
+            self._num_channels,
```

Comment:
Thanks, do you know which other extractors might need this kind of change?

Reply:
Most of them. I will address this in a future PR.